_slides/09-jared-sanders.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is a caption!
 
 ![codercat](https://octodex.github.com/images/codercat.jpg)
```

Comment:
If you want to change your image, you can grab a new one from here: https://octodex.github.com

Reply:
No thanks 👍